pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -852,22 +864,44 @@ def get_instancename(context, instancename, options, ns_names=None):
         Command-specific options from the command line (including --key and
         --namespace if specified).
 
-      ns_names (list of :term:`string` or None)
-        If None, this is a command with a single namespace.
-        If a list, this is a command with multiple namespaces
+      default_all_ns (:class:`py:bool`)
+        If True, the default is to return all namespaces.  Otherwise the
+        default is to return None
+
+      ns_names (list of :term:`string`, :term:`string`, or None)
+        * If None,  the detault namespace is to be used.
+        * if string, this is a command with a single namespace.
+        * If list, this is a command with multiple namespaces
 
     Returns:
 
       :class:`~pywbem.CIMInstanceName`: CIM instance path. It is never None.
+      Namespace included in instance name ONLY if it is in the parameter
+      instance name, or ns_names is None.
```

Comment:
You removed the `ns_names` parameter from the method.

Reply:
DONE. Comment modified to state that it includes namespace when in instance name or multiple namespaces specified in --namespace